<div class="py-4" >
    <div class="container" id="pdf-1">
        <div class="header">
            <img src="../../../../assets/LOGo.png" />
            <div class="header-sub" >
               <div>
                    <p>17115 Red Oak DR, Ste 123,</p>
                    <p>Houston TX 77090</p>
               </div>
               <div>
                    <p> P: 561.635.0392 | F: 832.4130059</p>
               </div>
               <div>
                    <p>CLIA: 45D2257672</p>
                    <p>Lab Director: Dr. Kerollos Shaker</p>
               </div>
            </div>
        </div>

        <div class="gradient-background" >

        </div>

        <div class="panel-header">
            <h4>Cancer Risk Panel</h4>
        </div>
  
      <div class="row box-conatiner">
        <div class="col-md-4">
          <div class="card">
            <div class="card-header">Patient</div>
            <div class="card-body">
              <p><strong>First Name:</strong> John</p>
              <p><strong>Last Name:</strong> Doe</p>
              <p><strong>Date of Birth:</strong> 01/01/1965</p>
              <p><strong>Gender:</strong> Male</p>
              <p><strong>Accession ID:</strong> 0123456789</p>
            </div>
          </div>
        </div>
  
        <div class="col-md-4">
          <div class="card">
            <div class="card-header">Specimen</div>
            <div class="card-body">
              <p><strong>Specimen Type:</strong> Buccal Swab</p>
              <p><strong>Collection Date:</strong> </p>
              <p><strong>Received Date:</strong> </p>
              <p><strong>Panel Coverage:</strong> 94.04%</p>
              <p><strong>Average Read Depth:</strong> 138x</p>
            </div>
          </div>
        </div>
  
        <div class="col-md-4">
          <div class="card">
            <div class="card-header">Ordering Physician</div>
            <div class="card-body">
              <p><strong>Physician:</strong> </p>
              <p><strong>Institution:</strong> Elixir DX</p>
              <p><strong>Reported Date:</strong> </p>
              <p><strong>Ref Accession:</strong> N/A</p>
              <p><strong>Ref Accession:</strong> N/A</p>
            </div>
          </div>
        </div>
      </div>
  
      <div class="alert-positive mt-4">
        Test Result: Pathogenic / Likely Pathogenic variant detected on CHEK2 gene.
      </div>
  
      <div class="result-key">
        <div class="result-key-box-cont" >
            <div class="result-key-box pathogenic"></div>
            <div>
                <p>POSITIVE: Pathogenic / </p>
                <p>Likely Pathogenic</p>
            </div>
        </div>
        <div class="result-key-box-cont" >
            <div class="result-key-box vus"></div>
            <div>
                <p>VUS: Variant of Unknown Significance</p>
                <p>Unknown Significance</p>
            </div>
        </div>
        <div class="result-key-box-cont"  >
            <div class="result-key-box negative" ></div>
            <div>
                <p>NEGATIVE: No clinically significant</p>
                <p>variants were detected</p>
            </div>
        </div>
      </div>

      <div style="margin-top: 30px;" >
        <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
          <thead>
            <tr>
              <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Regulatory Disclosures</th>
            </tr>
          </thead>
          <tbody>
            <tr style="background: #fff;" >
              <td style="border: 1px solid black; padding: 13px 20px;">
                Cancer risk factor screening is intended as a tool to guide physicians in the management of their patients and should not be treated as a diagnostic tool. NGS-based risk factor screening is considered a high-complexity laboratory-developed test (LDT) by Centers for Medicare & Medicaid Services (CMS) under the Clinical Laboratory Improvement Amendment (CLIA) and is not FDA-cleared. The test and performance metrics were validated in house by Elixir Diagnostics technical personnel (or designated scientific advisors) and approved by their Laboratory Director. The results are intended for use only by the ordering physician and/or designated healthcare provider. The ordering provider is responsible for 1) ascertaining the medical necessity of the ordered test, 2) resulting diagnoses, 3) management of the disease and/or decisions based on the data provided. Results rely on collection personnel following specified collection and shipment protocols. 
              </td>
            </tr>
          </tbody>
        </table>
        <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
          <thead>
            <tr>
              <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Test Limitations</th>
            </tr>
          </thead>
          <tbody>
            <tr style="background: #fff;" >
              <td style="border: 1px solid black; padding: 13px 20px;">
                The results of this screen are meant strictly to guide a physician in the management of their patient's health. Large genomic rearrangements, low frequency mutations, structural and somatic mutations are not reported. Presence of a pathogenic/likely pathogenic variant does not guarantee that an individual will develop cancer, nor is the absence of such variants a guarantee that an individual will not develop cancer. Any likely pathogenic or pathogenic variants detected in the report should be clinically correlated per physician’s advice and confirmed via the orthogonal testing platforms (Sanger Sequencing or PCR based tests).  
              </td>
            </tr>
          </tbody>
        </table>
  
        <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
          <thead>
            <tr>
              <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">References</th>
            </tr>
          </thead>
          <tbody>
            <tr style="background: #fff;" >
              <td style="border: 1px solid black; padding: 13px 20px;">
                  Richards, Sue, et al. "Standards and guidelines for the interpretation of sequence variants: a joint consensus
                      recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular
                      Pathology." Genetics in medicine 17.5 (2015): 405.Exome Aggregation Consortium et al. "Analysis of Protein-Coding
                      Genetic Variation in 60,706 Humans."Nature 536.7616 (2016): 285–291. PMC. Web. 13 May 2018. The 1000 Genomes
                      Project Consortium. "A Global Reference for Human Genetic Variation." Nature 526.7571 (2015): 68–74. PMC. Web. 13
                      May 2018.
              </td>
            </tr>
          </tbody>
        </table>
      </div>

      <div>
        <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
          <thead>
            <tr>
              <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Test Methodology</th>
            </tr>
          </thead>
          <tbody>
            <tr style="background: #fff;" >
              <td style="border: 1px solid black; padding: 13px 20px;">
                  Cancer Risk Factor Screening performed at Elixir Diagnostics utilizes next generation sequencing technology using IIIumina MiniSeq and NextSeq platforms. Genomic DNA is extracted from buccal swabs (Wet). The extracted DNA is tagemented and enriched for regions of interest using specific probes for each region outlined in the BED (Browser Extensible Data) file. Positive and negative controls are included in each run to validate amplicon preparation and targeted sequencing of coding regions, intronic/exonic boundaries, and variant detection. Sequences obtained are then aligned against a human reference genome and variants such as SNVs (Single Nucleotide Variants) and Indels (Insertions and Deletions) are noted. Computational analysis and variant calling are performed by Varsome (https://varsome.com/). Briefly, reads from the sequence output are aligned to the human reference genome (GRCh37) using the Burrows-Wheeler Aligner (BWA). Germline variants to reference are called using Sentieon's DNA Scope variant caller. The variants are annotated and filtered using the Elixir Diagnostics analysis work flow implementing the ACMG guidelines for interpretation of sequence variants. This includes comparison against the gnomAD population catalog of variants in 125,748 exomes, the 1000 Genomes Project Consortium, the NCBI ClinVar database of clinical assertions on variant’s pathogenicity and multiple lines of computational evidence on conservation and functional impact. The following databases and in-silico algorithms are used to annotate and evaluate the impact of the variant in the context of human disease: 1000 genomes, gnomAD, ClinVar, OMIM, dbSNP, VS PolyPhen2, PhyloP, GERP, and GHR Genes. Analysis is reported using the HGVS nomenclature (www.hgvs.org) as implemented by custom transcript annotation algorithm. Variants with an end ACMG classification of Pathogenic/Likely Pathogenic are reported as positive, while Benign/Likely Benign variants are reported as negative. Classification of pathogenicity is consistently updated by the National Institutes of Health and the information found within this report is consistent with the current knowledge base. All reports are reviewed prior to release by either Elixir Diagnostics Technical Supervisor or the General Supervisor. 
              </td>
            </tr>
          </tbody>
        </table>
  
        <table *ngIf="variantData.length > 0" class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
          <thead>
            <tr>
              <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Genes Evaluated</th>
            </tr>
          </thead>
          <tbody>
            <tr style="background: #fff;" >
              <td style="border: 1px solid black; padding: 13px 20px;">
                  {{variantData[0].genes}}
              </td>
            </tr>
          </tbody>
        </table>
      </div>
  
      <div *ngFor="let data of variantData">
        <h3 class="mt-4 bold" >{{data.Variant}}</h3>
        <table class="table table-striped table-custom mt-4 mb-0"  style="border: 1px solid black; border-collapse: collapse;">
          <thead>
            <tr>
              <th style="border: 1px solid black;" >Gene & Transcript</th>
              <th>Variant</th>
              <th style="border: 1px solid black;" >Inheritance</th>
              <!-- <th style="border: 1px solid black;" >Disorder or Phenotype</th> -->
              <th style="border: 1px solid black;" >Criteria</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td style="border: 1px solid black;" >{{data.GeneSymbol}}</td>
              <td style="border: 1px solid black;" >{{data.Variant}}</td>
              <td style="border: 1px solid black;" >{{data.Zygosity}}</td>
              <!-- <td style="border: 1px solid black;" >Li-Fraumeni syndrome 2</td> -->
              <td style="border: 1px solid black;" >{{data.ACMGRules}}</td>
            </tr>
          </tbody>
        </table>
    
        <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse;">
          <thead>
            <tr>
              <th style="border: 1px solid black;">Location</th>
              <!-- <th style="border: 1px solid black;">Allele State</th> -->
              <th style="border: 1px solid black;">Allelic Balance</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td style="border: 1px solid black;">{{data.ExonLocation}}</td>
              <!-- <td style="border: 1px solid black;">Heterozygous</td> -->
              <td style="border: 1px solid black;">{{data.AllelicBalance}}</td>
            </tr>
          </tbody>
        </table>
    
        <table class="table table-striped table-custom">
          <thead>
            <tr>
              <!-- <th style="border: 1px solid black;">Genomic Position</th> -->
              <th style="border: 1px solid black;"> Coverage</th>
              <th *ngIf="data.ACMGClassification !== 'Uncertain significance' && data.ACMGClassification !== 'Likely pathogenic' && data.ACMGClassification !== 'Pathogenic'" style="border: 1px solid black;">AcmgClassification</th>
              <th *ngIf="data.ACMGClassification === 'Likely pathogenic'" style="border: 1px solid black; background: orange;">AcmgClassification</th>
              <th *ngIf="data.ACMGClassification === 'Pathogenic'" style="border: 1px solid black; background: red;">AcmgClassification</th>
              <th *ngIf="data.ACMGClassification === 'Uncertain significance' " style="border: 1px solid black !important; background: yellow;">AcmgClassification</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <!-- <td style="border: 1px solid black;">Chr22:NC_000022.10:g.29095917C>G</td> -->
              <td style="border: 1px solid black;">{{data.Coverage}}</td>
              <td *ngIf="data.ACMGClassification !== 'Uncertain significance' && data.ACMGClassification !== 'Likely pathogenic' && data.ACMGClassification !== 'Pathogenic'" style="border: 1px solid black;">{{data.ACMGClassification}} {{data.ACMGRules}}</td>
              <td *ngIf="data.ACMGClassification === 'Likely pathogenic'" style="border: 1px solid black; background: orange;">{{data.ACMGClassification}} {{data.ACMGRules}}</td>
              <td *ngIf="data.ACMGClassification === 'Pathogenic'" style="border: 1px solid black; background: red;">{{data.ACMGClassification}} {{data.ACMGRules}}</td>
              <td *ngIf="data.ACMGClassification === 'Uncertain significance' " style="border: 1px solid black; background: yellow;">{{data.ACMGClassification}} {{data.ACMGRules}}</td>
            </tr>
          </tbody>
        </table>
        <div style="margin-top: 30px;" >
          <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
            <thead>
              <tr>
                <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Location</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td style="border: 1px solid black; padding: 13px 20px; background: transparent; color: rgb(255, 107, 107);">Gene Info</td>
              </tr>
            </tbody>
          </table>
    
          <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
            <thead>
              <tr>
                <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Gene Info</th>
              </tr>
            </thead>
            <tbody>
              <tr style="background: #fff;" >
                <td style="border: 1px solid black; padding: 13px 20px;">
                  The Variant is found at <span>{{data.HGVSNotation}}</span> location with a <span>{{data.VariantType}}</span> variant <span>{{data.HGVSCoding}}</span> change on the patient's <span>{{data?.GeneSymbol}}</span>.</td>
              </tr>
            </tbody>
          </table>
    
          <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
            <thead>
              <tr>
                <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Variant Interpretation</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td style="border: 1px solid black; padding: 13px 20px; background: transparent;">
                    <p>The missense variant NM_007194.4(CHEK2):c.917G>C (p.Gly306Ala) causes the same amino acid change as a
                        which is not likely to impact secondary protein structure as these residues share similar properties. 2 variants within 6
                        amino acid positions of the variant p.Gly306Ala have been shown to be pathogenic, while none have been shown to
                        be benign. For these reasons, this variant has been classified as Likely Pathogenic. </p>
                    <p class="mt-4" >Overview: OBSOLETE. Any Li-Fraumeni syndrome in which the cause of the disease is a mutation in the CHEK2 gene.</p>    
                </td>
              </tr>
            </tbody>
          </table>
      
          <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
            <thead>
              <tr>
                <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Inheritance</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td style="border: 1px solid black; padding: 13px 20px; background: transparent;">
                  {{ data.Inheritance ? data.Inheritance : '-' }}
                </td>
              </tr>
            </tbody>
          </table>
      
          <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
            <thead>
              <tr>
                <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">ACMG-Classification</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td style="border: 1px solid black; padding: 13px 20px; background: transparent; color: rgb(255, 107, 107);">
                  {{data.ACMGClassification}} {{data.ACMGRules}}
                </td>
              </tr>
            </tbody>
          </table>
      
          <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
            <thead>
              <tr>
                <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Gene & Disorder or Phenotype</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td style="border: 1px solid black; padding: 13px 20px; background: transparent;">Li-Fraumeni syndrome 2</td>
              </tr>
            </tbody>
          </table>
          <table *ngIf="data.gene_description" class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
            <thead>
              <tr>
                <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Gene Description</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td style="border: 1px solid black; padding: 13px 20px; background: transparent;">{{data.gene_description}}</td>
              </tr>
            </tbody>
          </table>

          <table *ngIf="data.pheno_type" class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
            <thead>
              <tr>
                <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Phenotype</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td style="border: 1px solid black; padding: 13px 20px; background: transparent;">{{data.pheno_type}}</td>
              </tr>
            </tbody>
          </table>
          
          <!-- <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
            <thead>
              <tr>
                <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">What’s next</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td style="border: 1px solid black; padding: 13px 20px; background: transparent;">Correlate the findings with clinical symptoms, biochemical profile and family history whilst closely monitoring the
                    subject with periodical visits.</td>
              </tr>
            </tbody>
          </table> -->
        </div>
      </div>
  
      <div class="footer">
        <p>CLIA: 45D2257672 &nbsp; | &nbsp; Lab Director: Dr. Kerollos Shaker</p>
      </div>
    </div>
    <div class="container">
      <button type="button" class="btn btn-info"  (click)="convertHtmlToPdf()" >EXPORT</button>
    </div>
  </div>
  
